prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java

```diff
@@ -137,6 +143,10 @@ public ProsperoConfig getProsperoConfig() {
         return new ProsperoConfig(channels);
     }
 
+    public List<Repository> getOverrideRepositories() {
```

Comment:
can we remove this? I don't think it's used anywhere

Reply:
Yes.